KTLOSNaxxLoot.json

```diff
@@ -551,7 +551,7 @@
 			"wowID":"23545"
 		},
 		"Eye of the Dead":{
-			"prio":["Lianli(1)","Claris(1)","Peku(2)","Soldeia(2)","Darkdwarf(2)","Healers EP/GP"],
+			"prio":["Lianli(1)","Claris(1)","Peku(2)","Soldeia(2)","Darkdwarf(2)","Thrane(1)","Jaz(1)","Healers EP/GP"],
```

Comment:
Thrane asked to be higher on this since he doesn't have rejuv gem nor shard.  The other healers ahead of him does.